cache/lru_cache.h

```diff
@@ -474,8 +473,8 @@ class LRUCache
 
  private:
   LRUCacheShard* shards_ = nullptr;
+  LRUCacheOptions options_;
```

Comment:
I may prefer to keep the secondary_cache_ pointer here. using options_.secondary_cache is hard to understand and maybe in the future, we can replace the secondary_cache_ on the fly (e.g., from local secondary to the remote secondary)

Reply:
Whether or not it is in the LRUCacheOptions or an independent argument, the rules would be the same for what you are suggesting.  As an LRUCacheOption member, it is easier to keep it with the rest of the options and treat it the same as everything else.

Reply:
Since the place to pass in the SecondaryCache might change, we may keep using options_.secondary_cache for now.